com.richwhitehouse.BigPEmu.yaml

```diff
@@ -0,0 +1,41 @@
+id: com.richwhitehouse.BigPEmu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: bigpemu
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.bigpemu_userdata
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - shared-modules/SDL2/SDL2-no-libdecor.json
+  - shared-modules/glu/glu-9.json
+
+  - name: bigpemu
+    buildsystem: simple
+    build-commands:
+      - install -Dm 0755 bigpemu ${FLATPAK_DEST}/bin/bigpemu
+      - install -Dm 0644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm 0644 bigpemu-icon.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - cp -r ReadMe.txt ${FLATPAK_DEST}/bin/
+      - cp -r Data ${FLATPAK_DEST}/bin/
+      - cp -r Scripts ${FLATPAK_DEST}/bin/
+      - cp -r Strings ${FLATPAK_DEST}/bin/
+      - cp -r plugins ${FLATPAK_DEST}/bin/
+    sources:
+      - type: archive
+        url: https://www.richwhitehouse.com/jaguar/builds/BigPEmu_Linux64_v113.tar.gz
```

Comment:
how will that work on other platforms?

Reply:
I added an `only-arches` clause, this is strictly x86_64 only

Reply:
This doesn't prevent a build on antyhing else.

Reply:
I added a flathub.json file to use `"only-arches": ["x86_64"]`, will that work?

Reply:
yes